dev-proxy/ConsoleLogger.cs

```diff
@@ -384,4 +340,46 @@ public object Clone()
             LogLevel = LogLevel
         }, _pluginEvents);
     }
+
+    /// <inheritdoc/>
+    public void Log<TState>(MSLogging.LogLevel logLevel, MSLogging.EventId eventId, TState state, Exception? exception, Func<TState, Exception?, string> formatter)
+    {
+        if (IsEnabled(logLevel))
+        {
+            var message = formatter(state, exception).ReplaceLineEndings();
+            switch (logLevel)
+            {
+                case MSLogging.LogLevel.Debug:
+                    LogDebug(message);
+                    break;
+                case MSLogging.LogLevel.Information:
+                    LogInfo(message);
+                    break;
+                case MSLogging.LogLevel.Warning:
+                    LogWarn(message);
+                    break;
+                case MSLogging.LogLevel.Error:
+                    LogError(message);
+                    break;
+            }
+        }
+    }
+
+    /// <inheritdoc/>
+    public bool IsEnabled(MSLogging.LogLevel logLevel) => LogLevel switch
+    {
+        LogLevel.Debug => true,
```

Comment:
This is confusing and at the first sight gives the impression that debug logging is always enabled which isn't the case. How could we clarify this?

Reply:
I renamed the property to CurrentLevel, and added comment